hack/testing/test-020-olm-upgrade.sh

```diff
@@ -65,13 +66,13 @@ fi
 
 
 # deploy EO
-os::cmd::expect_success 'deploy_marketplace_operator "openshift-operators-redhat" "elasticsearch-operator" "elasticsearch-operator" "true" '
+os::cmd::expect_success 'deploy_marketplace_operator "openshift-operators-redhat" "$previous_version" "elasticsearch-operator" "elasticsearch-operator" "true" '
```

Comment:
```
deploy_marketplace_operator(){
  local ns=$1
  local name=$2
  local channel=$3
  local package=${4:-$name}
  local global=${5:-false}
```
so you want `$name` to be `$previous_version`?

Reply:
wrong order